client/src/components/ReUsableComponents/RequiredButtons.tsx

```diff
@@ -0,0 +1,13 @@
+import Button, { ButtonProps } from '@mui/material/Button';
+import { purple } from '@mui/material/colors';
+import { styled } from '@mui/material/styles';
+
+const RequiredButton = styled(Button)<ButtonProps>(({ theme }) => ({
+  color: theme.palette.getContrastText(purple[500]),
+  backgroundColor: '#f14140',
+  '&:hover': {
+    backgroundColor: '#f14140',
+  },
```

Comment:
So, in the `theme` would've been best to set the primary color to the red color and then using `primary.main` for the background color. MUI will actually calculate dark and light red based on the main red given.

Reply:
This should be pretty easy to do just take a look at `client/themes/theme.ts` and change the hex code value.